src/snowflake/snowpark/mock/_functions.py

```diff
@@ -480,11 +565,11 @@ def mock_to_time(
     )
 
 
-@patch("to_timestamp")
-def mock_to_timestamp(
+def _to_timestamp(
     column: ColumnEmulator,
-    fmt: Optional[str] = None,
+    fmt: Optional[ColumnEmulator],
```

Comment:
with the change, it indicates that fmt can be a series of data instead of a single str.

Reply:
I've updated this type annotation. It should now be `ColumnOrLiteralStr`. I've also added tests that show the expected behavior for when fmt is a string or a column